src/Core/Console/Config.php

```diff
@@ -37,8 +37,8 @@
  *   set to the value of the last parameter. (e.g. "system loglevel 0" will
  *   disable logging)
  *
- * @author Tobias Diekershoff
- * @author Hypolite Petovan <mrpetovan@gmail.com>
+ * @author Tobias Diekershoff <tobias.diekershoff@gmx.net>
```

Comment:
That field is not the email address but the webby and is interpreted by the addon view in the admin panel as link to the profile of the author... So this is clearly wrong information there, as my profile is not lying at gmx.net

Reply:
Ok, forget it, we are not in the addon m) But why does this information now have to be in the file header? Is there any good reason?H ow much of the file did I really author?

Reply:
You authored the original CLI script that I later converted to a Console command, so you deserve to be credited.

This information is in the header because we can. It would be typically displayed in an automatically generated code documentation.